### PR TITLE
[9.4.x] jdbc string based config examples

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11169

### CODE REVIEW

File: documentation/src/main/asciidoc/user_guide/persistence_jdbc.adoc [hunk 1]
```diff
@@ -58,7 +58,7 @@ For detailed description of all the parameters used refer to the link:{javadocro
 [source,xml]
 ----
 <persistence>
-   <string-keyed-jdbc-store xmlns="urn:infinispan:config:store:jdbc:9.2" shared="true" fetch-state="false" read-only="false" purge="false">
+   <string-keyed-jdbc-store fetch-state="false" read-only="false" purge="false" key-to-string-mapper="org.infinispan.persistence.keymappers.DefaultTwoWayKey2StringMapper" dialect="H2">
```
Comment (oraNod) [inline documentation/src/main/asciidoc/user_guide/persistence_jdbc.adoc]: @ryanemerson looks like `key-to-string-mapper="org.infinispan.persistence.keymappers.DefaultTwoWayKey2StringMapper"` is the default so maybe we should remove it from the example?

Comment (ryanemerson) [reply documentation/src/main/asciidoc/user_guide/persistence_jdbc.adoc]: Yep. We can also remove `fetch-state`, `read-only` and `purge` as those attributes are also using the default values.

Comment (oraNod) [reply documentation/src/main/asciidoc/user_guide/persistence_jdbc.adoc]: @ryanemerson thanks. I removed those defaults from the declarative examples + ConfigurationBuilder. I also removed the footnote specific to derby. we dropped that in 10 and should probably do for 9.4 as well. ta.

**Comment by ryanemerson** [comment]
Merged, thanks @oraNod

**Pull request merged**